p2p/p2p.go

```diff
@@ -133,37 +133,17 @@ func (s *Service) Start() error {
 
 // start begins the p2p Service, including discovery. start does not terminate once called.
 func (s *Service) start(e chan error) {
-	if len(s.bootstrapNodes) == 0 && !s.noBootstrap {
+	if len(s.bootnodes) == 0 && !s.noBootstrap {
 		e <- errors.New("no peers to bootstrap to")
+		return
 	}
 
-	// this is in a go func that loops every minute due to the fact that we appear
-	// to get kicked off the network after a few minutes
-	// this will likely be resolved once we send messages back to the network
-	go func() {
-		for {
-			if !s.noBootstrap {
-				// connect to the bootstrap nodes
-				err := s.bootstrapConnect()
-				if err != nil {
-					e <- err
-				}
-			}
-
-			err := s.dht.Bootstrap(s.ctx)
-			if err != nil {
-				e <- err
-			}
-			time.Sleep(time.Minute)
-		}
-	}()
+	go s.bootstrap()
 
-	// Now we can build a full multiaddress to reach this host
-	// by encapsulating both addresses:
 	addrs := s.host.Addrs()
 	log.Info("You can be reached on the following addresses:")
 	for _, addr := range addrs {
-		log.Info(addr.Encapsulate(s.hostAddr).String())
+		fmt.Println("\t" + addr.Encapsulate(s.hostAddr).String())
```

Comment:
When I first read this line I thought why but after starting the node I actually quite like it indented.

Reply:
Yeah, I think it's a reasonable exception, just easier to see :P

Reply:
So much nicer to read :+1: